host/4/bullseye/amd64/dotnet/dotnet8-inproc/dotnet.Dockerfile

```diff
@@ -0,0 +1,59 @@
+# Build the runtime from source
+ARG HOST_VERSION=4.1033.5
+FROM mcr.microsoft.com/dotnet/sdk:8.0 AS runtime-image
+ARG HOST_VERSION
+
+ENV PublishWithAspNetCoreTargetManifest=false
+
+RUN BUILD_NUMBER=$(echo ${HOST_VERSION} | cut -d'.' -f 3) && \
+    git clone --branch feature/in-proc8 https://github.com/Azure/azure-functions-host /src/azure-functions-host && \
+    cd /src/azure-functions-host && \
+    HOST_COMMIT=$(git rev-list -1 HEAD) && \
+    dotnet publish -v q /p:CommitHash=$HOST_COMMIT src/WebJobs.Script.WebHost/WebJobs.Script.WebHost.csproj -c Release --output /azure-functions-host --runtime linux-x64 --framework net8.0 && \
```

Comment:
This is no longer self-contained by default. That is the reason that ASPNet is required.